_assets/stylesheets/pages/_seven-hills.scss

```diff
@@ -0,0 +1,11 @@
+.content-left {
+  @media screen and (min-width: $screen-md) {
+    padding-left: 4rem;
+  }
+}
+
+.content-right {
+  @media screen and (min-width: $screen-md) {
+  padding-right: 4rem;
+  }
+}
```

Comment:
Feels problematic that this file isn't actually scoped to the page and is using generic class names.

Reply:
Same could be said for pretty much all of these page CSS files. Unless I'm missing something.